lib/internal/test_runner/test.js

```diff
@@ -526,11 +526,11 @@ class Test extends AsyncResource {
     }
 
     const { args, ctx } = this.getRunArgs();
-    const after = runOnce(async () => {
```

Comment:
Why is `runOnce()` removed? Looking at the code below it looks like, `after()` could throw, end up in the `catch` and then run again.